Dockerfile.pai

```diff
@@ -1,11 +1,8 @@
-FROM ibmcom/powerai:1.6.0-base-ubuntu18.04-py3
+FROM ibmcom/powerai:1.6.1-base-ubuntu18.04-py3
 SHELL ["/bin/bash", "-c"]
 
 WORKDIR /workspace
 RUN mkdir assets
 
 COPY . .
-RUN cd /opt/anaconda3/bin && \
-        source activate base && \
-	pip install --upgrade pip &&\
-        pip install -r /workspace/requirements.txt
+RUN ["/bin/bash", "-c", "cd /opt/anaconda3/bin && source activate base && pip install --upgrade pip && pip install -r /workspace/requirements.txt" ]
```

Comment:
This seems to be there same command, any reason behind the change to the array version?

Reply:
Oh thank you, that's a good catch. So, the reason this change exists is because the SHELL command is not OCI compliant. An OCI compliant image was being asked for by the OpenShift team. If need be, I can grab them for a more in-depth explanation.

But! This change also requires getting rid of the actual shell command, which this commit didn't do. I'll make that change and recommit.

Reply:
ok, is this something we need to worry about in the normal dockerfile or just the power ai one? 

Also @MLnick @djalova @bdwyer2 you guys have spent time writing Dockerfiles could you confirm that this "change" doesn't change anything? As long as I get one other LGTM I'll merge this